includes/class-payment-information.php

```diff
@@ -222,4 +231,22 @@ public static function get_token_from_request( array $request ) {
 
 		return $token;
 	}
+
+	/**
+	 * Cchanges type type of the payment.
```

Comment:
Thanks for noticing this! I committed it in https://github.com/Automattic/woocommerce-payments/pull/979/commits/ebc9215635ce235899751a7bb75843c1b9511577 because there's an additional `the` needed.